nilearn/datasets/neurovault.py

```diff
@@ -2851,6 +2851,16 @@ def fetch_neurovault_motor_task(data_dir=None, verbose=1):
     """Fetch left vs right button press \
        group :term:`contrast` map from :term:`Neurovault`.
 
+    .. deprecated:: 0.11.2dev
+
+        This fetcher function will be removed in version>0.13.1
```

Comment:
```suggestion
        This fetcher function will be removed in version>0.13.1
```

Shouldn't we stick to 0.13.0 for all deprecations?

Reply:
We could but we have the 2 releases rule, so I was trying to be of for 0.11.1 + 2 releases -> 0.13.1